src/components/landing-page-body/landingPageBody.js

```diff
@@ -0,0 +1,134 @@
+      },
+    },
+    sectionsWrapper: {
+      borderBottom: "solid 0.5px",
+      borderWidth: "50%",
+      fontSize: "32px",
+      width: "62%",
+      margin: "0 auto",
+      opacity: "0.2",
+      marginTop: "65px",
+    },
+    sectionOneStyling: {
+      fontSize: "32px",
+      fontFamily: "'Century Gothic', sans-serif",
+    },
+    secondTypographyInSectionOne: {
+      marginTop: "25px",
+      fontFamily: "'Century Gothic', sans-serif",
+      fontSize: "23px",
+    },
+    viewProjectsButton: {
+      margin: "0 auto",
+      marginTop: "20px",
+      backgroundColor: ZEUS,
+      borderRadius: "0",
+      fontSize: "12px",
+      letterSpacing: "3px",
+      padding: "13px",
+      width: "185px",
+    },
+    sectionTwoStyling: {
+      fontSize: "30px",
+      marginTop: "15px",
+      fontFamily: "'Century Gothic', sans-serif",
+    },
+    sectionThreeStyling: {
+      fontSize: "15px",
+      marginTop: "25px",
+      fontFamily: "'Century Gothic', sans-serif",
+    },
+    weLoveToHearAboutYourProjectButton: {
+      border: "solid 2px",
+      margin: "0 auto",
+      marginTop: "40px",
+      borderRadius: "0",
+      fontSize: "12px",
+      letterSpacing: "3px",
+      padding: "13px",
+      width: "420px",
+      color: "black",
+      letterSpacing: "1.5px",
+      padding: "15px 15px",
+      fontWeight: "bald",
+    },
+  },
+});
+
+export default function LandingPageBody() {
+  return (
+    <ThemeProvider theme={theme}>
+      <Box
+        style={theme.custom.LandingPageBodyContainer}
+        sx={theme.custom.LandingPageBodyContainer.sx}
+      >
+        <Typography style={theme.custom.sectionOneStyling}>
+          Destination Eichler creates inviting and functional modern interiors
+          <Typography style={theme.custom.secondTypographyInSectionOne}>
+            SAN FRANCISCO BAY AREA & BEYOND
+          </Typography>
+          <Stack spacing={2} direction="row">
+            <Button style={theme.custom.viewProjectsButton} variant="contained">
+              VIEW PROJECTS
+            </Button>
+          </Stack>
+          <div style={theme.custom.sectionsWrapper}></div>
+        </Typography>
+        <Typography style={{ marginTop: "40px" }}>
+          <Typography style={theme.custom.sectionTwoStyling}>
+            LET'S WORK TOGETHER TO DESIGHN A JOYFUL..COLLESIVE, INSPIRING SPACE
+          </Typography>
+          <Typography style={theme.custom.sectionTwoStyling}>
+            _ ONE THATS A REFLECTION OF YOUR UNIQUE PERSONALITY, INTERSTS
+          </Typography>
+          <Typography style={theme.custom.sectionTwoStyling}>
+            AND DAY-TO-DAY NEEDS.
+          </Typography>
+          <div style={theme.custom.sectionsWrapper}></div>
+        </Typography>
+        <Typography style={{ marginTop: "40px" }}>
+          <Typography style={theme.custom.sectionThreeStyling}>
+            GHAITH KONBAZ DESIGHN IS A FULL-SERVICE INTERIOR DESIGHN FIRM BASED
+            IN GERMANY, BERLIN.
```

Comment:
All the hard coded text can be stored in an Obj/JSON.
Why?
many reasons, but some:
* in the future we might have **_a translation feature_** and having this obj of texts makes it easier to us to manipulate.
* if we wanted to store in the BE, it makes it easier for us to do less change in the code.
* we might have an admin route where the owner can have his custom inputs and change what's written (code free)